doc/classes/ShapeCast2D.xml

```diff
@@ -75,28 +75,28 @@
 			<return type="bool" />
 			<argument index="0" name="layer_number" type="int" />
 			<description>
-				Returns whether or not the specified layer of the [member collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.
+				Returns true if the specified layer of the [member collision_mask] is enabled, given a [code]layer_number[/code] between 1 and 32.
 			</description>
 		</method>
 		<method name="get_collision_normal" qualifiers="const">
 			<return type="Vector2" />
 			<argument index="0" name="index" type="int" />
 			<description>
-				Returns the normal containing one of the multiple collisions at [code]index[/code] of the intersecting object.
+				Returns the normal of one of the multiple collisions at [code]index[/code] of the intersecting object.
 			</description>
 		</method>
 		<method name="get_collision_point" qualifiers="const">
 			<return type="Vector2" />
 			<argument index="0" name="index" type="int" />
 			<description>
-				Returns the collision point containing one of the multiple collisions at [code]index[/code] at which the shape intersects the object.
+				Returns the collision point of one of the multiple collisions at [code]index[/code] where the shape intersects the colliding object.
 				[b]Note:[/b] this point is in the [b]global[/b] coordinate system.
 			</description>
 		</method>
 		<method name="is_colliding" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns whether any object is intersecting with the shape's vector (considering the vector length).
+				Returns true if any object is intersecting with the shape's vector (considering the vector length).
```

Comment:
Is there a grammatical problem with "Returns whether"? As pointed out above it's used quite a bit in the docs already.

If you want to use "Returns true", then "true" needs to be formatted as code ("Returns [code]true[/code]"). But IMO "whether" is better as it covers both true and false.

Reply:
I believe we should use "whether" consistently. As for it being a grammatical issue, I don't know. "true if" just seems like a more programmer thing to write.